packages/graphql-language-service/benchmark/fixtures/github.graphql

```diff
@@ -273,7 +273,7 @@ type BlameRange {
   startingLine: Int!
 }
 
-type Blob implements Node, GitObject {
+type Blob implements Node & GitObject {
```

Comment:
this is not valid syntax that fails in prettier v3

Reply:
indeed, this is not spec valid syntax, I'm not sure where the author got it from